tests/components/group/test_media_player.py

```diff
@@ -362,6 +420,17 @@ async def test_service_calls(hass):
     await hass.async_block_till_done()
     assert hass.states.get("media_player.kitchen").state == STATE_OFF
 
+    calls = async_mock_service(hass, MEDIA_DOMAIN, SERVICE_MEDIA_STOP)
```

Comment:
We want to check that the service call reaches the grouped entities. So we can't mock out the service.

Reply:
I failed to create a working test. Implementing `stop` support for demo players is easier. There is a PR: https://github.com/home-assistant/core/pull/50485 I will push a fixed test after PR merge.

Reply:
Done